test/__snapshots__/fixtures.test.ts.snap

```diff
@@ -6,62 +6,6 @@ exports[`fixtures lint code blocks should work as expected: code-blocks.md 1`] =
     "column": 1,
     "endColumn": 6,
     "endLine": 3,
-    "fix": {
```

Comment:
`eslint-plugin-markdown` is required to be upgraded first? We're locking `unified@^11.0.4` for now.